src/dropdown/dropdown.component.ts

```diff
@@ -96,7 +98,11 @@ export class Dropdown implements OnInit, AfterContentInit, OnDestroy {
 	 * @memberof Dropdown
 	 */
 	@Input() type: "single" | "multi" = "single";
-
+	/**
+	 * Set to `true` to apply the light style to the dropdown.
+	 * @memberof Dropdown
+	 */
+	@Input() light = false;
```

Comment:
We decided to standardize this and go instead with 
```suggestion
	@Input() theme: "light" | "dark" = "dark";
```

This will become a pattern we use across the library. Theme `enum` will be the type.